src/core/thread/network_data_tlvs.hpp

```diff
@@ -634,6 +638,50 @@ class BorderRouterEntry
      */
     void SetOnMesh(void) { mFlags |= kOnMeshFlag; }
 
+    /**
+     * This method indicates whether or not the Nd-Dns flag is set.
+     *
+     * @retval TRUE   If the Nd-Dns flag is set.
+     * @retval FALSE  If the Nd-Dns flag is not set.
+     *
+     */
+    bool IsNdDns(void) const { return (mReserved & kNdDnsFlag) != 0; }
```

Comment:
We should rename `mReserved`.

Reply:
will update.

Reply:
Will update. Any suggestion on the name? @jwhui